php/blocks/class-loader.php

```diff
@@ -100,6 +100,25 @@ public function editor_assets() {
 				$this->enqueue_block_styles( $block['name'], array( 'preview', 'block' ) );
 			}
 		}
+
+		// Used to conditionally show notices for blocks belonging to an author.
+		$author_blocks = get_posts(
+			array(
+				'author'         => get_current_user_id(),
+				'post_type'      => 'block_lab',
+				// We could use -1 here, but we'll limit it just in case somebody goes nuts with making custom blocks.
+				'posts_per_page' => 99,
+			)
+		);
+
+		$author_block_slugs = array_map(
```

Comment:
This seemed to work locally, though `array_map()` worked as well:

```php
$author_block_slugs = wp_list_pluck( $author_blocks, 'post_name' );
```

Reply:
I always forget about `wp_list_pluck`. _Much_ nicer - thanks!

Reply:
Fixed in edc702e431aedddab45225732e52289461de8636

Reply:
Yeah, functions like that hard to keep track of. I have to look them up every time I think about using them.